go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/algorand/avm-abi v0.2.0
 	github.com/algorand/go-algorand v0.0.0-20220211161928-53b157beb10f
 	github.com/algorand/go-algorand-sdk v1.22.1-0.20221209152459-223656f08456
```

Comment:
Do we still need v1?

Reply:
I don't think we do but I'd rather save that for another PR.

Reply:
Sure, sounds fine.